plugins/windows/src/main/java/net/java/games/input/RawInputEventQueue.java

```diff
@@ -62,6 +67,18 @@ public final void start(List<RawDevice> devices) throws IOException {
 			throw queue.getException();
 	}
 
+	/**
+	 * Interrupts the threads that are reading the joystick input.
+	 */
+	public void destroyAll() {
+        this.threads.forEach(thread -> {
+			// Must be done in this order!
```

Comment:
Might be useful to have a multi-line comment explaining why they need to be done in this order, maybe with links to any relevant docs.

The function name is also a bit vague. At a glance, I wouldn't be sure what it's destroying. Maybe `destroyAllQueueThreads` or `destroyAllThreads`?

Reply:
It makes sense if you consider how the different conditions play out but I've added some more docs to hopefully save people time. 

Changed it to `destroyAllThreads`, that does work a lot better 🥳